### DISCUSSION TITLE
Ssd1963 speedup

### DISCUSSION THREAD

**Discussion by moreauftheobald** [Bodmer/TFT_eSPI#1719] [Q&A - General] [answered]
Hello, 

I was wondering if it was possible to speeder thé clock offre ssd1963 insta 8 bit parallel mode ? 

I've sein PLL paramèters in the  SSD1963_Init.h à line 3 to 6 incluses, but how could it bé modifier to speedup thé clock raté ? 

For information, m'y display israéliennes plugged to an esp32 clocked à 240 MHz. 

Regards

**Comment by Bodmer** [comment]
I am not sure what you wish to achieve by changing clock speeds within the display.  Note that increasing the PLL parameters will not change the speed that the processor can write to the screen.

**Comment by moreauftheobald** [comment, marked as answer]
i just want to make the screen displaying faster. like changing the speed on spi port, but with 8bit parralel port.

**Reply by Bodmer** [reply]
The parallel interface used is working at the maximum rate for a GPIO pin. So the speed will be 80Mbps to the display. This is a limitation of the ESP32. Processors such as the RP2040 can write much faster at 250Mbps (about 3 times faster). So one approach is to use an ESP32 for WiFi and RP2040 to drive the display.

Minimising the screen areas that are updated via software approaches is effective. The viewport functions may help.

**Reply by moreauftheobald** [reply]
ok thanks, i understand now how it works.

My display is working quite faste (around 150 ms for a full screen) that's enought for me, i was juste wondering if it was possible to get a little bit faster.

thanks a lot for your help.
